subprojects/packagefiles/cmark-gfm/meson.build

```diff
@@ -0,0 +1,106 @@
+project(
+    'cmark-gfm',
+    'c',
+    version: '0.29.0.gfm.6',
+    default_options: ['warning_level=3'],
+)
+
+cdata = configuration_data(
+    {
+        'HAVE_STDBOOL_H': true,
+        'HAVE___BUILTIN_EXPECT': true,
+        'HAVE___ATTRIBUTE__': true,
+    },
+)
```

Comment:
In the cmake build, these are defined as:
```cmake
CHECK_INCLUDE_FILE(stdbool.h HAVE_STDBOOL_H)
CHECK_C_SOURCE_COMPILES(
  "int main() { __builtin_expect(0,0); return 0; }"
  HAVE___BUILTIN_EXPECT)
CHECK_C_SOURCE_COMPILES("
  int f(void) __attribute__ (());
  int main() { return 0; }
" HAVE___ATTRIBUTE__)
```

I am not sure it makes sense to hardcode these...

Reply:
Fixed.